basic-features.md

```diff
@@ -128,7 +128,7 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 
```

Comment:
```suggestion
| [暂停/恢复 DDL](/ddl-introduction.md#ddl-相关的命令介绍) | E | N | N | N | N | N | N | N | N |
| [DDL 分布式并行执行框架](/tidb-distributed-execution-framework.md) | E | N | N | N | N | N | N | N | N |
```

Reply:
@benmaoer PTAL

Reply:
It is recommended not to add a distributed framework as it is an implementation approach rather than the user's DDL definition language.